src/screens/mentee-home/mentee-home.tsx

```diff
@@ -63,10 +67,14 @@ const MenteeHome: React.FC = () => {
       .then(() => navigation.navigate(AppRoute.OPPORTUNITY_DETAILS));
   };
 
+  if (!user) {
```

Comment:
If there is no user data I think we should perform a logout

Reply:
This is just for type checks. There always will be user data.